projects/grass-addons/project.yaml

```
@@ -0,0 +1,23 @@
+name: grass-addons
+language: cpp
+primary_contact: "neteler@gmail.com"
```

```suggestion
primary_contact: "neteler@osgeo.org"
```

However, I'd suggest to point here to https://grass.osgeo.org/support/community/